examples/muscle_driven_with_contact/contact_forces_inequality_constraint_muscle.py

```diff
@@ -40,23 +37,6 @@ def prepare_ocp(biorbd_model_path, phase_time, n_shooting, min_bound, max_bound)
     dynamics = DynamicsList()
     dynamics.add(DynamicsFcn.MUSCLE_DRIVEN, with_residual_torque=True, with_contact=True, expand=False)
 
-    # Constraints
-    constraints = ConstraintList()
-    constraints.add(
-        ConstraintFcn.CONTACT_FORCE,
-        min_bound=min_bound,
-        max_bound=max_bound,
-        node=Node.ALL,
-        contact_force_idx=1,
-    )
-    constraints.add(
-        ConstraintFcn.CONTACT_FORCE,
-        min_bound=min_bound,
-        max_bound=max_bound,
-        node=Node.ALL,
-        contact_force_idx=2,
-    )
-
     # Path constraint
     n_q = biorbd_model.nbQ()
```

Comment:
I am not sure I see where this was moved. Is this just remove? If so, why did not just you changed it for TRACK_CONTACT_FORCE?

Reply:
I literally copied the following example which contains contact: maximum_predicted_height_CoM.py. There is no contact constraints associated with this example. Finally, I did not guess from TRACK_CONTACT_FORCE Function that it would replace CONTACT_FORCE.